config/ksql-server.properties

```diff
@@ -17,7 +17,8 @@
 
 ### HTTP  ###
 # The URL the KSQL server will listen on:
-listeners=http://0.0.0.0:8088
+# The default is any IP4 or IP6 interface on the machine.
+listeners=http://0.0.0.0:8088,http://[::]:8088
```

Comment:
There's another occurrence of `0.0.0.0:8088` below in this file (in the comment about HTTPS). Perhaps worth updating that comment too?

Reply:
I figured that one was there purely to show to to use SSL.  I didn't think it needed an IPv6 version too.